### PR TITLE
promo design review v2

### PR DESCRIPTION
**Pull request by dprzybylek**
* more place for text
* 20px margin on the right of img

### CODE REVIEW

File: packages/design-system/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livechat/design-system",
-  "version": "0.9.0",
+  "version": "0.9.1-beta.4",
```
Comment (kamilmateusiak) [inline packages/design-system/package.json:3]: bump to `0.9.1-beta.5`

Comment (dprzybylek) [reply packages/design-system/package.json:3]: sure sure

File: packages/design-system/src/components/Promo/style.scss [hunk 2]
```diff
@@ -92,7 +92,7 @@ $base-class: 'promo';
     }
 
     .#{$base-class}__wrapper {
-      max-width: 520px;
+      max-width: 680px;
```
Comment (kamilmateusiak) [inline packages/design-system/src/components/Promo/style.scss:95]: Banner should be changed as well. The same max-width value was there and I expect the same issue with spacings

Comment (dprzybylek) [reply packages/design-system/src/components/Promo/style.scss:95]: done, jo approved

**Pull request merged**